src/main/webapp/staticFeeds/sidebar.json

```diff
@@ -1,7 +1,7 @@
 {"sidebar" :  
             [
-                {"title": "Home", "hyperlink" : "/", "target" : "", "faIcon" : "fa-home", "displayFlag" : ""},
                 {"title": "Beta Settings", "hyperlink" : "settings", "target" : "", "faIcon" : "fa-cog", "displayFlag" : "sidebarShowSettings"},
-                {"title": "Log out", "hyperlink" : "/portal/Logout", "target" : "", "faIcon" : "fa-sign-out", "displayFlag" : ""}
+                {"title": "External URL", "hyperlink" : "http://www.wisc.edu", "target" : "", "faIcon" : "fa-external-link", "displayFlag" : ""},
```

Comment:
Ah, I wasn't clear.  It should have a `target` of `_blank`, because it is functioning as an example for downstream (frame-based-app) developers, and the UX pattern in MyUW is that `href`s to external URLs use `target="_blank"`.  *should* in the `RFC` sense of the word, as it were.